ros2_control_libraries/Dockerfile

```diff
@@ -38,3 +38,7 @@ RUN sudo pip3 install ./control-libraries/python
 RUN sudo rm -rf ./control-libraries
 
 RUN sudo apt-get clean && sudo rm -rf /var/lib/apt/lists/*
+
+FROM ${BASE_IMAGE}:${BASE_TAG} as final
+RUN sudo rm -rf /usr/include/eigen3
+COPY --from=control-libraries-build /usr/ /usr/
```

Comment:
I am not sure I understand what is going here. Are you not still in the `control-libraries-build` layer?

Reply:
ok nevermind I missed the `FROM` line 42. All good now.

Reply:
Could there be some conflicts between lib installed in the base ubuntu image and the ros image used line 42? How does the copy handles duplicates? Is that a recursive merge?

Reply:
I guess there could but we are only copying `/usr/local` from the ubuntu image of control libraries so I think it's unlikely that something happens there. Not sure what type of merge it is.